src/sentry/static/sentry/app/components/group/seenInfo.jsx

```diff
@@ -76,54 +76,76 @@ class SeenInfo extends React.Component {
       projectId,
     } = this.props;
     return (
-      <dl className="seen-info">
-        <dt key={0}>{t('When')}:</dt>
+      <DateWrapper>
         {date ? (
-          <dd key={1}>
+          <TooltipWrapper>
             <Tooltip title={this.getTooltipTitle()} disableForVisualTest>
-              <TimeSince className="dotted-underline" date={date} />
+              <IconInfo size="xs" color="gray500" />
```

Comment:
Do you need this icon? It seems like the primary information and doesn't need additional decoration. 🤷

Reply:
Agreed. I’ll ditch it.